Documentation/ApiOverview/Seo/CanonicalApi.rst

```diff
@@ -16,27 +16,32 @@ cHash.
    EXT:seo. You can find information about how to install and use it in the
    :doc:`EXT:seo manual <ext_seo:Index>`.
 
-Excluding arguments from the generation
-=======================================
+.. _canonicalapi-additionalparameters:
 
-TYPO3 will fallback to building a URL of current page and appending query strings.
-It is possible to exclude specific arguments from being appended.
+Including specific arguments from the generation
+================================================
+
+TYPO3 will building a URI of the current page and append query strings
+which are needed for the cHash calculation (vital arguments to uniquely identify
+the given content URI).
+
+It is possible to additionally include specific arguments.
```

Comment:
Let's add the "This is especially important with for example detail pages of records. The query parameters are crucial to show the right content." here for clarification.